.gitignore

```diff
@@ -19,3 +19,4 @@ appengine/build
 tools/shows-gen/yt-api-key.json
 src/jekyll/_includes/svgs/
 .bundle/
+google_appengine/
```

Comment:
@PaulKinlan @petele Just in case this won't surprise you. Basically let git ignore AppEngine directory. I think this is relevant.